docs/source/extensions/callbacks.rst

```diff
@@ -216,12 +216,6 @@ on_configure_sharded_model
 .. automethod:: pytorch_lightning.callbacks.Callback.on_configure_sharded_model
     :noindex:
 
-on_before_accelerator_backend_setup
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. automethod:: pytorch_lightning.callbacks.Callback.on_before_accelerator_backend_setup
-    :noindex:
-
```

Comment:
Don't remove this from the docs. Instead add to the docstring that this hook is deprecated, and it'll render on the docs site